opcua/ua/uatypes.py

```diff
@@ -629,7 +629,6 @@ def to_binary(self):
     def from_binary(data):
         obj = ExtensionObject()
         obj.TypeId = NodeId.from_binary(data)
-        obj.Encoding = uabin.Primitives.UInt8.unpack(data)
         if obj.Encoding & (1 << 0):
```

Comment:
what is that? you cannot test for the value of Encoding without first unpacking?????